src/org/exist/xquery/ExternalModuleImpl.java

```diff
@@ -270,6 +270,11 @@ public boolean moduleIsValid(DBBroker broker) {
         return mSource != null && mSource.isValid(broker) == Source.VALID;
     }
 
+    @Override
+    public void reset(XQueryContext context) {
```

Comment:
How about adding an `@deprecated` annotation? to the interface and `@Deprecated` javadoc?